pkg/wolfssl/doc.txt

```diff
@@ -0,0 +1,184 @@
+/**
```

Comment:
this doc file probably needs an update: the dtls-wolfssl application is not listed here. (and maybe some other applications like the wolfssl_crypto_test will be removed)

Reply:
Doc updated.